x-pack/plugins/security_solution/public/flyout/document_details/shared/components/misconfiguration_insight.tsx

```diff
@@ -54,6 +59,14 @@ export const MisconfigurationsInsight: React.FC<MisconfigurationsInsightProps> =
     pageSize: 1,
   });
 
+  useEffect(() => {
+    uiMetricService.trackUiMetric(
+      METRIC_TYPE.COUNT,
+      `${MISCONFIGURATION_INSIGHT} instance id: ${dataTestSubj}`
```

Comment:
I wouldn't couple the test id with the tracking tbh, let's introduce a prop dedicated to tracking. If we change the test id for some reason, the tracking info will also change which might affect reports and numbers

Reply:
makes sense. @CohenIdo would we also want to insert the field (user/host) to the id? so we can track which is more used?

Reply:
yes.

Reply:
I've added a `telemetrySuffix` prop. left the values as plain texts rather than constants because if this component will be moved i prefer it to be clear that the suffix might need to change as well.